src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -37,9 +37,23 @@
                     </div>
                 </div>
             </template>
+            <div v-if="showRunButton" id="run-form-group" class="form-group row">
+                <div class="col-sm-2"></div>
+                <div class="col-sm-6">
+                    <button @click.prevent="runReport" class="btn" type="submit"
+                            :disabled="disableRun ? 'disabled' : undefined">
```

Comment:
This should do the right thing, especially if it's a boolean:
```suggestion
                            :disabled="disableRun">
```